langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiChatModelIT.java

```diff
@@ -116,8 +108,12 @@ void should_call_function_with_argument(String deploymentName, String gptVersion
 
         ToolSpecification toolSpecification = ToolSpecification.builder()
                 .name(toolName)
-                .description("Get the current weather")
-                .parameters(getToolParameters())
+                .description("Get the current temperature, in celsius or fahrenheit depending on the city location")
```

Comment:
I wouls keep old API in these tests to make sure we are backwards compatible with the old tool API.
Testing of the new API is happening in `AzureOpenAiAiServicesWithToolsIT`